docs/current_docs/manuals/developer/snippets/functions/arguments-contextual/typescript/index.ts

```diff
@@ -0,0 +1,25 @@
+import { object, func, argument, Directory, File } from "@dagger.io/dagger"
+
+@object()
+class MyModule {
+  @func()
+  async repoFiles(
```

Comment:
The module and repo examples seem almost the same. Why not have a single one? Also maybe consider calling the argument `sourceDir`, since that would probably be the most common use case?

Reply:
Because by using tabs, I can actually explain how to declare a contextual value, and the setup is different for each language but yeah I would love to put all things in common. Do you have any suggestion of an example I could create to make that happen?

Reply:
Sorry if I wasn't clear. The tabs are fine. I meant that the moduleFiles() and repoFiles() fn signatures seem almost identical, why not merge them into a single example? Also also change the argument name to sourceDir, which may make it easier to grok?

Reply:
Because there would be 2 different argument in the functions but yeah I could merge them

Reply:
Fixed in new example